apps/stress-test/src/components/v9/StressContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import * as React from 'react';
+import { injectGlobalCss } from '../../shared/injectStyles';
+import { getTestParams } from '../../shared/testParams';
+import { performanceMeasure } from '../../shared/performanceMeasure';
+import { StressComponent } from './StressComponent';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)',
+    ...shorthands.gap('10px'),
+  },
+});
+
+export type StressContainerProps = {
+  numChildren?: number;
+};
+
+export const StressContainer: React.FC<StressContainerProps> = ({ numChildren = 10 }) => {
+  const [checked, setChecked] = React.useState(false);
+
+  if (getTestParams().test === 'mount') {
+    performanceMeasure('stress', 'start');
+  } else if (getTestParams().test === 'inject-styles') {
+    setTimeout(() => {
+      performanceMeasure('stress', 'start');
+      injectGlobalCss('');
+    }, 2000);
+  } else if (getTestParams().test === 'prop-update') {
+    /* eslint-disable-next-line react-hooks/rules-of-hooks */
+    React.useEffect(() => {
+      setTimeout(() => {
+        performanceMeasure('stress', 'start');
+        setChecked(true);
+      }, 2000);
+    }, []);
+  }
+
+  const styles = useStyles();
+
+  const kids = new Array(numChildren).fill('1');
+  return (
+    <div className={styles.container}>
+      {kids.map((_, index) => {
+        return <StressComponent key={index} id={index.toString()} checked={checked} />;
```

Comment:
why do we have index.toString() in v9 but not v8?

Reply:
I've updated v8 to also call `.toString()` since `id` should be a string. Not sure why v8 didn't catch the TS error.